src/NUnitFramework/tests/Attributes/TestOrderAttributeTests.cs

```diff
@@ -22,6 +24,51 @@ public void CheckOrderIsCorrect()
             Assert.AreEqual(work.Children[1].Test.Name, "Y_SecondTest");
             Assert.AreEqual(work.Children[2].Test.Name, "Z_ThirdTest");
         }
-        
+
+        [Test]
+        [TestCaseSource(nameof(Cases))]
+        public void CheckClassOrderIsCorrect(List<Type> candidateTypes)
+        {
+            var testSuite = TestBuilder.MakeFixture(candidateTypes);
+
+            var work = TestBuilder.PrepareWorkItem(testSuite, null) 
+                as CompositeWorkItem;
```

Comment:
Haha good point.  I typically strive to keep lines of code at a length 80 characters or shorter, which is what I was going for here.  But that may have harmed readability a little here.